configs/desktop_savant_events_schemas.json

```diff
@@ -0,0 +1,1122 @@
+            "meta-cat-2": "'bookmark'"
+          },
+          "schema": {
+            "$schema": "http://json-schema.org/schema#",
+            "type": "object",
+            "properties": {
+              "timestamp": {
+                "type": "number",
+                "minimum": 0
+              },
+              "category": {
+                "type": "string",
+                "enum": [
+                  "savant"
+                ]
+              },
+              "method": {
+                "type": "string",
+                "enum": [
+                  "bookmark"
+                ]
+              },
+              "object": {
+                "type": "string",
+                "enum": [
+                  "save"
+                ]
+              }
+            },
+            "required": [
+              "timestamp",
+              "category",
+              "method",
+              "object"
+            ]
+          }
+        },
+        {
+          "name": "remove bookmark",
+          "description": "client removes a bookmark",
+          "amplitudeProperties": {
+            "meta-cat-1": "'feature'",
+            "meta-cat-2": "'bookmark'"
+          },
+          "schema": {
+            "$schema": "http://json-schema.org/schema#",
+            "type": "object",
+            "properties": {
+              "timestamp": {
+                "type": "number",
+                "minimum": 0
+              },
+              "category": {
+                "type": "string",
+                "enum": [
+                  "savant"
+                ]
+              },
+              "method": {
+                "type": "string",
+                "enum": [
+                  "bookmark"
+                ]
+              },
+              "object": {
+                "type": "string",
+                "enum": [
+                  "remove"
+                ]
+              }
+            },
+            "required": [
+              "timestamp",
+              "category",
+              "method",
+              "object"
+            ]
+          }
+        },
+        {
+          "name": "open hamburger menu",
+          "description": "client opens the hamburger menu",
+          "amplitudeProperties": {
+            "meta-cat-1": "'feature'"
+          },
+          "schema": {
+            "$schema": "http://json-schema.org/schema#",
+            "type": "object",
+            "properties": {
+              "timestamp": {
+                "type": "number",
+                "minimum": 0
+              },
+              "category": {
+                "type": "string",
+                "enum": [
+                  "savant"
+                ]
+              },
+              "method": {
+                "type": "string",
+                "enum": [
+                  "hamburger_menu"
+                ]
+              }
+            },
+            "required": [
+              "timestamp",
+              "category",
+              "method"
```

Comment:
According to the [telemetry event definition](https://dxr.mozilla.org/mozilla-central/source/toolkit/components/telemetry/Events.yaml#206) this should also have an object called "open"

Reply:
Su asked in the bug about whether it's necessary to provide the full coordinates, or just enough to uniquely identify the event.

If there are no other events with category=savant and method=hamburger_menu, there's no functional difference here from specifying the object, or am I misunderstanding?

Reply:
There aren't *now*, but the definitions could change under us – I see no disadvantage to being explicit, and an advantage of some safety